CMakeLists.txt

```diff
@@ -71,29 +88,74 @@ message(
 
 # Option to build OMNI Compiler as a submodule. Should be ON as CLAW uses a
 # specific version of OMNI Compiler at the moment.
-option(BUILD_OMNI "Build OMNI Compiler" ON)
 option(OMNI_ENABLE_GNU_INTRINSIC "OMNI Compiler - Enable GNU intrinsic extensions" ON)
 
 if(OMNI_ENABLE_GNU_INTRINSIC)
   list(APPEND OMNI_CONF_OPTION "--enable-gnu-extension")
 endif()
 
+add_custom_target(create_int_install_dir ALL
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${INT_CLAW_HOME})
+
+install(
+    DIRECTORY ${INT_CLAW_HOME}/
+    DESTINATION ${CMAKE_INSTALL_PREFIX}
+    FILE_PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ)
+install(CODE "execute_process(COMMAND ${Python_EXECUTABLE} ${CMAKE_INSTALL_PREFIX}/etc/${CLAW_PY_CONF_FILE})")
```

Comment:
Why using Python here? Honestly that's a dependency you do not need.

Reply:
what would be a better way to execute this command ?

Reply:
Just not using Python for the configuration. I don't see the point.

Reply:
Well, one other dependency you really don't need in _java_ application is a driver out of  bash scripts. For a number of reasons, I'd like to get rid of them. However, rewriting them in pure java, is too time-consuming. So I needed an easy, powerful and readable script-language that could be compiled to JVM bytecode. That's where python comes along (via Jython). 

However, the amount of changes was growing to big, so I decided to do that in the next PR.  Only I already, replaced 1 script with python by that time, and it was too much trouble to revert it back. Hence as a quick hack, it currently dumps all variables into bash script (at build time).

Reply:
Well change like this should be discuss with the community first (even if it small right now). Otherwise, just make the project private and get rid of all the community guidelines we discussed when we made the claw-project.